ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -581,7 +581,9 @@ public void accountNameMismatchTest() throws Exception {
   @Test
   public void oldStyleUserMetadataTest() throws Exception {
     ByteBuffer content = ByteBuffer.allocate(0);
-    BlobProperties blobProperties = new BlobProperties(0, "userMetadataTestOldStyleServiceID");
+    BlobProperties blobProperties =
+        new BlobProperties(0, "userMetadataTestOldStyleServiceID", Account.UNKNOWN_ACCOUNT_ID,
```

Comment:
Some tests don't test any of the blob props and hence have used the UNKNOWN accountId and containerId. For those that matters, have generated random ids.